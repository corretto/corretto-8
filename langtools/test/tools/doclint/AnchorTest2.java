/*
 * @test /nodynamiccopyright/
 * @bug 8020313
 * @summary doclint doesn't reset HTML anchors correctly
 * @build DocLintTester
 * @run main DocLintTester -ref AnchorTest2.out AnchorTest2.java AnchorTest2a.java
 * @run main DocLintTester -ref AnchorTest2.out AnchorTest2a.java AnchorTest2.java
 */

/** */
public class AnchorTest2 {
    /** <a name="AnchorTest2"> </a> */
    public void a_name_AnchorTest2() { }

    /** <a name="AnchorTest2"> </a> */
    public void a_name_AnchorTest2_already_defined() { }

    /** <a name="AnchorTest2a"> </a> */
    public void a_name_AnchorTest2a_defined_in_other_file() { }
}
