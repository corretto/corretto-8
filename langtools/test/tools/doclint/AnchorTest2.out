AnchorTest2.java:15: error: anchor already defined: AnchorTest2
    /** <a name="AnchorTest2"> </a> */
           ^
1 error
